src/components/app-content-switcher/AppContentSwitcher.jsx

```diff
@@ -0,0 +1,49 @@
+import { Typography } from '@mui/material'
+import Tooltip from '@mui/material/Tooltip'
+import Switch from '@mui/material/Switch'
+import Stack from '@mui/material/Stack'
+
+const AppContentSwitcher = ({ isStudent, handleChange, tooltipRight, tooltipLeft, spacing, leftText, rightText, colorActive, colorInActive, typographyVariant, styles }) => {
```

Comment:
naming, isStudent used for checked prop and activeStyles, it should be called active or checked to be reusable

Reply:
refactored